crates/onboarding/src/theme_preview.rs

```diff
@@ -22,6 +25,18 @@ pub struct ThemePreviewTile {
     style: ThemePreviewStyle,
 }
 
+fn child_radius() -> Pixels {
+    static CHILD_RADIUS: OnceLock<Pixels> = OnceLock::new();
```

Comment:
Nit, but we could use `LazyLock` there instead. That'd simplify a thing or two I bet.